app/main/views.py

```diff
@@ -517,3 +520,116 @@ def mousereport_pdf(animal_id):
+                  'day_three': int(form['daythree_check'].data),
+                  'euthanized': int(form['euthanized_check'].data), 'checkup_notes': form['notes'].data}
+        try:
+            fexperiment.SurgeryStatus.insert1(tuple_)
+            flash("Surgery status for animal {} on date {} updated.".format(animal_id, date, tuple_['day_one']))
+        except Exception as ex:
+            ex_message = "An exception of type {} occurred. More information:\n\n{}".format(type(ex).__name__,
+                                                                                            ex.args)
+            flash(ex_message)
+            flash(tuple_)
+        return redirect(url_for('main.surgery_status'))
+    if len(fexperiment.SurgeryStatus & key) > 0:
+        data = (fexperiment.SurgeryStatus & key).fetch(order_by='timestamp DESC')[0]
+        return render_template('surgery_edit_status.html', form=form, animal_id=data['animal_id'],
+                               date=data['date'], day_one=bool(data['day_one']), day_two=bool(data['day_two']),
+                               day_three=bool(data['day_three']), euthanized=bool(data['euthanized']),
+                               notes=data['checkup_notes'])
+    else:
+        return render_template('404.html')
+
+
+@main.route('/api/v1/surgery/notification', methods=['GET'])
+def surgery_notification():
+    fexperiment = dj.create_virtual_module('csmith_testing', 'csmith_testing')
+    num_to_word = {1: 'one', 2: 'two', 3: 'three'}
+    slack_notification_channel = "#slack_api_testing"
+    slack_manager = "cameron.smith"
+    slacktable = dj.create_virtual_module('pipeline_notification', 'pipeline_notification')
+    domain, api_key = slacktable.SlackConnection.fetch1('domain', 'api_key')
+    slack = Slacker(api_key, timeout=60)
+    if len(fexperiment.Surgery - fexperiment.SurgeryStatus) > 0:
+        missing_data = (fexperiment.Surgery - fexperiment.SurgeryStatus).proj().fetch()
+        for entry in missing_data:
+            fexperiment.SurgeryStatus.insert1(entry)
+    surgery_data = (fexperiment.Surgery & {'surgery_outcome': 'Survival'}).fetch()
+    for entry in surgery_data:
+        if (datetime.date.today() - entry['date']).days < 4:
+            status = (fexperiment.SurgeryStatus & entry).fetch(order_by="timestamp DESC")[0]
+            day_key = "day_" + num_to_word[(datetime.date.today() - entry['date']).days]
+
+            edit_url = "<{}|Update Status Here>".format(url_for('main.surgery_update',
+                                                                _external=True,
+                                                                animal_id=entry['animal_id'],
+                                                                date=entry['date']))
+            if (status['euthanized'] == 0 and status[day_key] == 0):
+                manager_message = "{} needs to check animal {} for {} surgery on {}. {}".format(entry['username'].title(),
```

Comment:
if you want you could use  a formatted print `'{username} needs to check animal {animal_id} for {surgery_type} surgery on {date}. {edit_url}'.format(**entry, edit_url=edit_url)`